src/main/java/com/faforever/client/leaderboard/LeaderboardServiceImpl.java

```diff
@@ -62,4 +65,10 @@
         throw new IllegalArgumentException("Not supported: " + ratingType);
     }
   }
+
+  public CompletableFuture<List<Player>> getPlayerObjectsById(@NotNull String id)
+  {
+    System.out.println(id);
```

Comment:
Never do a system out print line

Reply:
If needed, then aaa @Slf4J to the class and use log.debug(...)

Reply:
I think they know that. I suspect that being a left over